packages/untp-test-suite/src/constant/index.ts

```diff
@@ -0,0 +1 @@
+export * from './importMeta.constant';
```

Comment:
Where is the `importMeta.constant` file located? I can't see any file in this directory except for the `index.ts` file. I think you just forgot to push the `importMeta.constant.ts` file.

Reply:
I think you can merge the next branch into this branch, and then you can utilize the `getCurrentFilePath` and `getCurrentDirPath` functions there. It can be similar to the file you are missing.